src/encoded/audit/experiment.py

```diff
@@ -4478,57 +4480,98 @@ def check_experiment_atac_encode4_qc_standards(experiment, files_structure):
+                                peaks_report['idr_rep_peaks_file'] = f['@id']
+                                if metric['reproducible_peaks'] > 70000:
+                                    peaks_report['idr_rep_peaks_qual'] = 'pass'
+                                if metric['reproducible_peaks'] <= 70000 and metric['reproducible_peaks'] >= 50000:
+                                    peaks_report['idr_rep_peaks_qual'] = 'warning'
+                                if metric['reproducible_peaks'] < 50000:
+                                    peaks_report['idr_rep_peaks_qual'] = 'not_compliant'
+
+                        # Replicate concordance is only reported for replicated experiments
+                        if 'replication_type' in experiment and \
+                                experiment['replication_type'] != 'unreplicated':
+                            detail = (
+                                f'According to ENCODE4 standards, peaks files in replicated '
+                                f'ATAC-seq assays processed by the uniform processing pipeline '
+                                f'should have a rescue ratio and self-consistency ratio < 2. '
+                                f'Having only one of these ratios < 2 is acceptable. '
+                                f'{audit_link(path_to_text(f["@id"]),f["@id"])} '
+                                f' has a rescue ratio of {rescue:.2f} and self-consistency ratio '
+                                f'of {self_consistency:.2f}.'
+                                )
+                            if rescue >= 2 and self_consistency >= 2:
+                                yield AuditFailure('insufficient replicate concordance', detail, level='NOT_COMPLIANT')
+                            elif rescue >= 2 or self_consistency >= 2:
+                                yield AuditFailure('borderline replicate concordance', detail, level='WARNING')
+
+    # The better reproducible peaks value in overlap or IDR thresholded peaks is reported
+    detail_items = []
+    if 'overlap_rep_peaks' in peaks_report.keys():
+        detail_items.append(tuple((str(peaks_report['overlap_rep_peaks']), peaks_report.get('overlap_rep_peaks_file'))))
+    if 'idr_rep_peaks' in peaks_report.keys():
+        detail_items.append(tuple((str(peaks_report['idr_rep_peaks']), peaks_report.get('idr_rep_peaks_file'))))
+    audit_peaks = ' and '.join(m[0] for m in detail_items)
+    file_links = ', '.join(audit_link(path_to_text(m[1]), m[1]) for m in detail_items)
+    detail = (
+        f'According to ENCODE4 standards, ATAC-seq assays processed by the uniform processing '
+        f'pipeline should have either >150k reproducible peaks in an overlap peaks file, or '
+        f'>70k in an IDR thresholded peaks file. 100-150k or 50-70k peaks respectively is '
+        f'acceptable, and <100k or <50k respectively is not compliant. '
+        f'File(s) {file_links} have {audit_peaks} peaks.')
+    if 'pass' in peaks_report.values():
```

Comment:
So in case you have multiple files you will have them listed? will it be clear which is which? I think that it would make more sense to report them separately? (I think you can yield directly form lines 4514-4536) ?

Reply:
After we talked, the yield is still outside the loop (because all files need to be checked first), but the detail now has an output type descriptor after each ENCFF. This also led me to see that I needed a variable to track files that had been flagged with multiple QC and avoid yielding if any had issues. I also added a couple more comments to make the intent clearer